packages/nx/src/command-line/generate.ts

```diff
@@ -145,6 +169,13 @@ async function promptForCollection(
     return customCollection
       ? `${customCollection}:${generatorName}`
       : generator;
+  } else if (deprecatedChoices.size > 0) {
+    throw new Error(
```

Comment:
Let's do this:
1. Warn them there are no non-deprecated
2. Let them choose which deprecated one they want
3. Run it